keyboards/kk/65/info.json

```diff
@@ -0,0 +1,104 @@
+{
+    "manufacturer": "Kory Hicks",
+    "keyboard_name": "KK65",
+    "maintainer": "apricity-spam",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D2", "D1", "D0", "D3", "D6", "C6", "B0", "C7", "F7", "E6", "F0", "F1", "F4", "F5", "F6"],
+        "rows": ["D7", "B4", "B5", "B6", "B7"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x4B4B"
+    },
+    "layouts": {
+        "LAYOUT_68_ansi": {
```

Comment:
If this matches the 68_ansi community layout, it should enable that.

Reply:
I'm not sure what would be considered matching. The community 68_ansi is the same number of keys but has a slightly different physical layout. I'd be happy to add it if it is considered matching or compatible.